cosmos/providers/dbt/core/operators.py

```diff
@@ -184,9 +194,9 @@ def add_global_flags(self):
             if global_flag_value is not None:
                 if isinstance(global_flag_value, dict):
                     # handle dict
-                    dict_string = json.dumps(global_flag_value)
+                    yaml_string = yaml.dump(global_flag_value)
                     flags.append(dbt_name)
-                    flags.append(f"'{dict_string}'")
+                    flags.append(yaml_string)
```

Comment:
why did this change from json to yaml?

Reply:
good question! This is what the CLI help docs are expecting. When I changed to the executable path -- it also broke as a dict string.